README.md

```diff
@@ -61,26 +61,26 @@ Access to your inverter data can be acheived by connecting the inverter's RS485
 
 ## Change Log
 v1.4.9:<br>
-Update H3 registers by cross-reference to other inverter models. Addresses to be confirmed are marked tbc in 'modbusH3_RS485_LAN.yaml'
+Update H3 registers by cross-reference to other inverter models. Addresses to be confirmed are marked tbc in 'ModbusH3_RS485_LAN.yaml'
 
 v1.4.8:<br>
 Fix errors in 'template.yaml' and 'templateH1G2.yaml' affecting Battery Warranty Remaining and Battery Life Remaining.<br>
-Improve handling of unknonw values in 'template.yaml' and 'templateH1G2.yaml'.<br>
+Improve handling of unknown values in 'template.yaml' and 'templateH1G2.yaml'.<br>
 Setting for the number of days to keep moved to 'secrets.yaml'. See [note](https://github.com/TonyM1958/HA-FoxESS-Modbus/issues/20) for info on configuring this.
 
 v1.4.6:<br>
-Updates to 'modbusH1G2_RS485..yaml', 'templateH1G2.yaml' and 'viewsH1G2_sensor.yaml'.
+Updates to 'ModbusH1G2_RS485..yaml', 'templateH1G2.yaml' and 'viewsH1G2_sensor.yaml'.
```

Comment:
change of case in a filename

Reply:
Also fixing double-dot that I just noticed.